mm2src/coins/nft.rs

```diff
@@ -331,26 +338,30 @@ where
         ..
     }) = coins.get_mut(ticker)
     {
+        // Todo: We should be storing/fetching NFT lists per address, by having a separate table for each address.
```

Comment:
@mariocynicys If this is finished before the db refactor, it will be multiple tables in the HD wallet unified db (rmd160 from mm2 internal derivation path) but NFT data gets fetched completely if it's a new db so data will just be fetched in the new dbs in each address dir (db path)

Reply:
cool.
i would say we should have a single table with address as a column in it instead though.
having addresses in table names doesn't look right (not tickers, but i know we already did that).

Reply:
> i would say we should have a single table with address as a column in it instead though.

It makes sense actually and it wouldn't cause any problems when using address based paths for each address in the future as this redundant data in db can be useful anytime.

Reply:
The only problem can be backward compatibility though, but I don't think it's a case here as we can just add a column through migrations. Will look into this.